src/Ushahidi/Modules/V5/Common/Errors.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Common;
+
+use Exception;
+use Ushahidi\Core\Exception\NotFoundException;
+use Ushahidi\Core\Exception\ValidatorException;
+use Ushahidi\Core\Exception\AuthorizerException;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+trait Errors
```

Comment:
Please don't use traits for that. It's a pointless abstraction a little bit and will make reviewing the code later much easier. Keep it stupid simple, even if that means Extending NotFoundException, ValidatorException itd. This trait does not bring much value imho